src/binding/julia/defs.hpp

```diff
@@ -0,0 +1,221 @@
+#ifndef DEFS_HPP
+#define DEFS_HPP
+
+#include "openPMD/openPMD.hpp"
+
+#include <jlcxx/array.hpp>
+#include <jlcxx/jlcxx.hpp>
+#include <jlcxx/module.hpp>
+#include <jlcxx/stl.hpp>
+#include <jlcxx/tuple.hpp>
+#include <jlcxx/type_conversion.hpp>
+
+#include <algorithm>
+#include <array>
+#include <complex>
+#include <map>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+using namespace openPMD;
+
+template <typename T> using array7 = std::array<T, 7>;
+
+// Generate code fo all openPMD types. Use is e.g. as follows:
+//   #define USE_TYPE(NAME, ENUM, TYPE) \
```

Comment:
Some compilers warn when you put a backslash at the end of a line comment, since this extends the comment (often unintentionally) to the next line. Can you use a `/* block comment */` for this to avoid that?

Reply:
Done.